src/pages/atomicdex/api/v20-dev/non_fungible_tokens/index.mdx

```diff
@@ -0,0 +1,762 @@
+        "metadata": "{\"name\":\"Forest Mushrooms\",\"description\":\"Sweet and nutty with distinct earthy tones, these forest mushrooms will make a fine addition to all your soups, salads, and potions.\",\"image\":\"https://metadata.thesmurfssociety.com/ingredients/nft/5.Forest_Mushrooms.jpg\",\"animation_url\":null,\"external_url\":\"https://app.thesmurfssociety.com/detail/ingredients/5\",\"attributes\":[{\"trait_type\":\"Type\",\"value\":\"Ingredient\"},{\"trait_type\":\"Category\",\"value\":\"COMMON\"}]}",
+        "last_token_uri_sync": "2022-12-13T13:12:50.840Z",
+        "last_metadata_sync": "2023-05-15T07:30:04.882Z",
+        "minter_address": "ERC1155 tokens don't have a single minter",
+        "possible_spam": false,
+        "uri_meta": {
+          "image": "https://metadata.thesmurfssociety.com/ingredients/nft/5.Forest_Mushrooms.jpg",
+          "token_name": "Forest Mushrooms",
+          "description": "Sweet and nutty with distinct earthy tones, these forest mushrooms will make a fine addition to all your soups, salads, and potions.",
+          "attributes": [
+            {
+              "trait_type": "Type",
+              "value": "Ingredient"
+            },
+            {
+              "trait_type": "Category",
+              "value": "COMMON"
+            }
+          ],
+          "animation_url": null
+        }
+      }
+    ],
+    "skipped": 1,
+    "total": 1
+  },
+  "id": null
+}
+```
+</CollapsibleSection>
+
+
+## Get a list of NFT transfers {{label : 'get_nft_transfers', tag : 'API-v2'}}
+
+Returns a list of the NFT transfers involving the user, shown in descending order of the `block_timestamp` value of the NFT's last transfer.
+
+
+### Request Parameters
+
+| Parameter     | Type             | Description                                                                                               |
+| ------------- | ---------------- | ----------------------------------------------------------------------------------------------------------|
+| chains        | array            | List of chains to scan for NFTs.                                                                          |
+| max           | boolean          | Optional, defaults to `false`. If `true`, response will return all NFTs without pagination, and will ignore the `limit` and `page_number` values.               |
+| limit         | integer          | Optional, defaults to `10`. The number of NFTs displayed per page in response.                            |
+| page_number   | integer          | Optional, defaults to `0`. The page offset for items in response.                                         |
+| filters       | object           | Optional. A standard [TokenFilter](/atomicdex/api/v20#token-filter) object.                               |
+
+
+### Response Parameters
+
+| Parameter            | Type             | Description                                                                                      |
```

Comment:
[updated info about transfers](https://gist.github.com/laruh/248d652d88891f3cadb09fea3a441466#get_nft_transfers-request)

same as for nft object `struct NftTransferCommon` will be flattened in response.
Updates: some fields have became optional.
`token_uri` field was added in nft transfers response. its the same field as from Nft object. 
`collection_name` is the same field as external `name` field from Nft object.